versions.props

```diff
@@ -1,15 +1,15 @@
 ch.qos.logback:logback-classic = 1.1.7
 # catch-all jackson version
-com.fasterxml.jackson.*:jackson-* = 2.9.9
+com.fasterxml.jackson.*:jackson-* = 2.10.1
```

Comment:
Release notes for jackson 2.10 look reasonably non-scary: https://github.com/FasterXML/jackson/wiki/Jackson-Release-2.10.1